src/ec/curve25519/x25519.rs

```diff
@@ -63,7 +63,10 @@ fn x25519_public_from_private(
     let private_key: &[u8; SCALAR_LEN] = private_key.bytes_less_safe().try_into()?;
     let private_key = ops::MaskedScalar::from_bytes_masked(*private_key);
 
-    #[cfg(all(not(target_os = "ios"), target_arch = "arm"))]
+    #[cfg(all(
+        not(any(target_os = "ios", target_os = "visionos")),
+        target_arch = "arm"
```

Comment:
I don't think we need this change. nobody is going to be trying to build for 32-bit ARM VisionOS, right?